.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
+Summarize the Frictionless Data JSON error output and if there are any errors, 
+send a summary message to a given Slack channel. Send a more detailed 
+error summary to std out and create a detailed txt file that can be accessed via GitHub
+artifacts.
+
+To run elsewhere, define SLACK_WEBHOOK in your repo's secrets and change the 
```

Comment:
These instructions aren't quite clear to me in terms of what I'd need to do if I wanted to set this up myself —  maybe because some of the instructions belong at a higher level? (Not sure where that would be.)

Reply:
I agree that placing the documentation higher would make more sense, but obviously, that's not ideal for this repo. For now I've just ported some of the yml blocks to the doc string